Products overhaul (#77)
pretix/pretix-docs: docs/guides/products/sessions.md

If your event's schedule is large, you have deviating prices for some of the sessions, or each session can only be attended by a limited number of people, you should use [add-on products](sessions.md#option-b-add-on-products-with-fixed-time-slots). 

If you have a large schedule with deviating prices and you also have sessions with varying start and end times that cannot be neatly organized into time slots because of overlap, you should use the [agenda constraints plugin](sessions.md#option-c-add-on-products-with-variable-time-slots). 

If you are not planning sessions that all take place within the scope of one big main event, but a series of events that can be attended individually, refer to our article on [event series](../event-series.md) instead. 

## Option A: Questions

If you are only offering a limited number of sessions and are not faced with a realistic risk of overcrowding in any of them, you can create a multiple-choice question. 
Customers will be prompted to answer this question while placing their order. 

> This has the upside of making it easy for users to change their mind later on. 

Reviewer: > This has the upside of making it easy for users to change their mind later on. 

This could also be achieved with add-on-products via "order changes" in the cancellation settings of the event.
Of course this depends on how add-on-products are utilized apart from sessions. We might have to test this to be sure.

Your event may be made up of several smaller events ("sessions") with more limited capacities, with attendees being able to pick and choose which of these sessions they want to attend. 
If you are planning a conference with multiple workshops, a celebration with a variety of activities, or a similar event, you may need a way to control access to those sessions. 

pretix offers you two methods to do this. 
You can either use add-on products with fixed time slots, or a plugin with advanced functions for variable time slots. 

> This article will guide you through both of them. 

Reviewer 2: ```suggestion
This article will guide you through both methods. 
```

Mir sind ein bisschen zu viele Pluralsubstantive zwischen dem letzten "methods" und diesem "both of them".
